### DISCUSSION TITLE
1.6.0

### DISCUSSION THREAD

**Discussion by patrikjuvonen** [multitheftauto/mtasa-blue#3069] [General]
After two years of feature development, bug fixing, and bug making; the MTA Team is delighted to announce the full release of MTA San Andreas 1.6.0! 🎉

Thank you to all beta testers for the past couple of months of feedback and bug reports!

## Release highlights

This update is meant to focus on fixes and changes, that were held back for a few releases in the past due to their compatibility-breaking nature, rather than new features, but there are already a lot of features planned for the next release!

Here's a list of highlights for this release:

- Many high FPS related inconsistency issues have been fixed by @Merlin 
- Script support for custom IMG containers, and ability to set model flags. Thanks to @TheNormalnij 
- A number of graphical effects
    - Added support for vehicle sun glare effect. Thanks to @gta191977649 and @TheNormalnij 
    - Added corona rain reflections. Thanks to @lopezloo 
    - Added big sun lens flare effect. Thanks to @gta191977649 
    - Added dynamic ped shadows. Thanks to @lopezloo 
    - Grass should now render correctly. Thanks to @TFP-dev 
- ARM support for MTA server is here! This should still considered experimental. Thanks to @botder 
- Added new Default 2023, GWEN Blue and GWEN Orange GUI skins. Thanks to @Haxardous 
- Added missing GTA special character skins. Thanks to @Allerek 
- Pictures taken with the camera weapon are now saved in high quality. Thanks to @lopezloo 
- Many stability improvements
- Many synchronization improvements
- Many varying size fixes, quality of life improvements, updates and security enhancements to both core and resources!
- Updates to all language translations from our Crowdin

If you are a scripter, you really should wanna dig in to these improvements! Show off your creations on our #showroom channel on our Discord server and have a chance at getting your work featured on our social media pages.

Read [the news post](https://forum.multitheftauto.com/topic/140935-multi-theft-auto-san-andreas-16-is-released/) on our forums and [1.6 release notes](https://wiki.multitheftauto.com/wiki/Changes_in_1.6) on Wiki for a complete set of changes and more information about the improvements listed above.

## Backwards compatibility

1.6 is not compatible with the older MTA:SA versions (including the previous release - MTA:SA 1.5.9). This means that you need to install the 1.6 client if you wish to play on 1.6 servers. There were a bunch of technical reasons for this incompatibility and they were all necessary for us to continue improving the mod.

This release includes 12 backwards incompatible changes and 5 deprecations which you should be aware of. Please check out our release notes on our Wiki site for more information.

## Important notice to Windows 7 and Windows 8.x users

If you are using Windows 7 or 8.x, please upgrade your system to Windows 10 or 11 as soon as possible. Windows 7 and 8.x are no longer supported by Microsoft (since January 2020 and January 2023 respectively) and most software (including Google Chrome and Steam) which means you are running an insecure system. Multi Theft Auto will also eventually drop Windows 7 and 8.x support sometime in the future, so it would be a good idea to start looking at upgrade options right now. Thank you!

CEF in MTA is no longer updated for Windows 7 or 8.x. This is because CEF no longer supports those versions of Windows. This bad for security, so please upgrade to Windows 10+ and MTA to 1.6+

We currently provide two variants of the MTA client, one for Windows 10+ and one for Windows 7/8.x and you can find the download links also conveniently on our home page by pressing the Download button, which reveals a dropdown with the two options.

![Screenshot 2023-06-16 233125](https://github.com/multitheftauto/mtasa-blue/assets/22572159/f3959b4c-3c75-4d3a-a96a-5c294c4d205c)

## What happens to 1.5?

1.5 installations will be kept intact even after the 1.6 release, to let players and server owners switch over to the new version whenever they feel they are ready. However, new features and changes will only be available on 1.6.

Note, that you can have both 1.5 and 1.6 installed at the same time on your computer to allow you to switch between if you want. MTA will prompt you to switch to the correct version if the server doesn't support your current version.

## Finally

- please consider donating @ https://multitheftauto.com/donate
- and don't forget to join our Discord server @ https://discord.com/invite/mtasa

A big thank you to all who donated, contributed and tested MTA 1.6.0, without all of you, this release would not have been possible ❤️

Please read our [1.6 release announcement](https://forum.multitheftauto.com/topic/140935-multi-theft-auto-san-andreas-16-is-released/) for more information.

You can download 1.6 from our home page at https://multitheftauto.com/ or wait for the automatic update to prompt you to update.

You can also find the latest binaries from [nightly.multitheftauto.com](https://nightly.multitheftauto.com/) and [linux.multitheftauto.com](https://linux.multitheftauto.com/).

Don't forget to update your server resources! Latest resources can be found from https://mirror-cdn.multitheftauto.com/mtasa/resources/

<hr /><em>This discussion was created from the release <a href='https://github.com/multitheftauto/mtasa-blue/releases/tag/1.6.0'>1.6.0</a>.</em>

**Comment by Hamorri227** [comment]
a Trojan was detected when the file was scanned for viruses
<img width="1166" height="868" alt="image" src="https://github.com/user-attachments/assets/dd9e4c31-2b73-41ab-a248-4bd1d3573e33" />

**Comment by Xenius97** [comment]
> a Trojan was detected when the file was scanned for viruses

False positive detection, you can check source code doesn't contain any malicious.
VBA32 is a russian noname non-trustable antivirus.
